frontend/package.json

```diff
@@ -14,11 +14,6 @@
     },
     "dependencies": {
         "@bcgov-nr/nr-fsa-theme": "^1.1.0",
-        "@fortawesome/fontawesome-svg-core": "^6.2.1",
-        "@fortawesome/free-regular-svg-icons": "^6.2.1",
-        "@fortawesome/free-solid-svg-icons": "^6.2.1",
-        "@fortawesome/vue-fontawesome": "^3.0.2",
-        "@popperjs/core": "^2.11.6",
```

Comment:
I'm not sure where we use this @popperjs/core package previously, just want to make sure we tested everything still works after removed it, thanks!!

Reply:
PopperJS is for tooltip and pop-ups, we are not using anywhere in the new design.